### PR TITLE
Support decimals in user configurable reports

### PR DESCRIPTION
**Pull request by NoahCarnahan**
Along with https://github.com/dimagi/fluff/pull/97, adds support for decimal type indicators in user configurable reports.
@czue @nickpell

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by NoahCarnahan** [comment]
Just realized I'm missing tests...

**Comment by czue** [comment]
you can add a test, though the type handling and tests leave something to be desired at the moment (related to this comment: https://github.com/dimagi/commcare-hq/pull/5275/files#diff-6e61c59b51d45a149319af5f04233351R61)

**Comment by czue** [comment]
(let me know if you want to block merge on tests)

**Comment by NoahCarnahan** [comment]
Thanks for pointing that out. I'll write a quick test.

File: corehq/apps/userreports/tests/test_data_source_config.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import json
 import os
 import datetime
+from decimal import Decimal, getcontext
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_data_source_config.py]: F401 'getcontext' imported but unused

File: corehq/apps/userreports/tests/test_pillow.py [hunk 2]
```diff
@@ -54,4 +54,7 @@ def _check_sample_doc_state(self):
             self.assertEqual(1, rows.rowcount)
             row = rows.fetchone()
             for k, v in row.items():
-                self.assertEqual(expected_indicators[k], v)
+                if isinstance(expected_indicators[k], decimal.Decimal):
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_pillow.py]: F821 undefined name 'decimal'

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by NoahCarnahan** [comment]
@czue added tests

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
:+1:

**Pull request merged**